conf/ocsci/infra_nodes.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
do we need a new conf file or we can use conf/ocsci/tainted_worker_nodes.yaml and update it?

Reply:
@ebenahar ,
Earlier I used the same file you mentioned, but after discussion with @petr-balogh , we arrived at a separate conf file because infra count was different in tainted_worker_nodes.yaml and not sure what are all the tests depends on it. Moreover this has default count of 6 workers on top of which we will be adding 3 more infra nodes using machinesets. 

If you still feel this file will be sufficient then I can revert back to tainted_worker_nodes.yaml . WDYT ?

Reply:
I am ok with either way, so we can have the new file